### PR TITLE
Add failing test fixture for `StaticDataProviderClassMethodRector`

### PR DESCRIPTION
**Pull request by OskarStark**
# Failing Test for StaticDataProviderClassMethodRector

Based on https://getrector.com/demo/3074e792-c0f4-4db7-bdd4-fb2c4b55bee3

### CODE REVIEW

**Comment by TomasVotruba** [comment]
This should be doable, as we know about the parent class.
Please, don't fix anything manually yet... :D I'll check it today.

File: tests/Rector/Class_/StaticDataProviderClassMethodRector/Fixture/test_case.php.inc [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace PHPUnit\Framework;
+
+namespace Rector\Tests\PHPUnit\Rector\Class_\StaticDataProviderClassMethodRector\Fixture;
+
+class TestCase
```
Comment (OskarStark) [inline tests/Rector/Class_/StaticDataProviderClassMethodRector/Fixture/test_case.php.inc:7]: Not sure this works, I think you check for this namespace in rules, right?

Comment (TomasVotruba) [reply tests/Rector/Class_/StaticDataProviderClassMethodRector/Fixture/test_case.php.inc:7]: Yeah, it should be spit 1 class = 1 file, so PHPStan can autoload it propperly.
But don't worry about it :)

Comment (OskarStark) [reply tests/Rector/Class_/StaticDataProviderClassMethodRector/Fixture/test_case.php.inc:7]: Would be cool if one could "Add new file" like for gists.github.com on the demo page

Comment (TomasVotruba) [reply tests/Rector/Class_/StaticDataProviderClassMethodRector/Fixture/test_case.php.inc:7]: I think that might lead people to commiting /vendor here :D 

Also, does Github link allow multiple files?

Comment (OskarStark) [reply tests/Rector/Class_/StaticDataProviderClassMethodRector/Fixture/test_case.php.inc:7]: > Also, does Github link allow multiple files?

I don't know

**Pull request closed without merging**